core/service/config/default_1.toml

```diff
@@ -211,3 +204,15 @@ crsgen = 100
 preproc = 25000
 keygen = 1000
 reshare = 1
+
+# Internal threading config (Tokio + Rayon pool sizes).
+# Must be the same across all parties to avoid diverging behavior.
+# If unset, defaults derive from available CPUs.
+# Defaults:
+#   num_tokio_threads = ceil(#CPUs / 8)
+#   num_rayon_threads = #CPUs - num_tokio_threads
+# Recommended to also set threshold.num_sessions_preproc = 1.5 * #CPUs.
+# Example for a 16‑core machine:
```

Comment:
Maybe highlight that it is recommended to have the default be equal to the amount of available cores?

Reply:
added in https://github.com/zama-ai/kms/pull/232/commits/53f5c3d641fb6b07144cb179b95a53ed28022653